content/en/community/hackathons/sessions/baby-steps/sol/04-store-strings/main.c

```diff
@@ -55,7 +55,7 @@ int main(int argc, char* argv[])
     int listenfd = -1;
     struct sockaddr_in serv_addr;
 
-    int filefd = open("my_strings", O_WRONLY | O_CREAT | O_TRUNC, 0644);
+    int filefd = open("my_strings.txt", O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
I think we should rename `filefd` to `file_fd` and `listenfd` to `listen_fd` to match the rest of the code?